async-openai/src/client.rs

```diff
@@ -29,15 +29,23 @@ pub const API_BASE: &str = "https://api.openai.com/v1";
 /// Name for organization header
 pub const ORGANIZATION_HEADER: &str = "OpenAI-Organization";
 
-impl Client {
-    /// Create client with default [API_BASE] url and default API key from OPENAI_API_KEY env var
-    pub fn new() -> Self {
+impl Default for Client {
+    // Create client with default [API_BASE] url and default API key from OPENAI_API_KEY env var
```

Comment:
```suggestion
    /// Create client with default [API_BASE] url and default API key from OPENAI_API_KEY env var
```
Lets preserve the comment for [doc](https://doc.rust-lang.org/rust-by-example/meta/doc.html) as well